Code_Exercises/Asynchronous_Execution/source.cpp

```diff
@@ -11,16 +11,10 @@
  * ~~~~~~~~~~~~~~~~~~~~
  *
  * // Default construct a queue
- * auto q = sycl::queue{};
+ * sycl::queue queue q = sycl::queue{};
```

Comment:
Small nitpick but do you want people to use the namespace or explicitly write sycl:: each time? Up to you but could be worth making that clear during the exercise.